spec/services/search/postgres/tag_spec.rb

```diff
@@ -14,7 +14,7 @@
       result = described_class.search_documents(tag.name)
 
       expect(result.first.keys).to match_array(
-        %w[id name hotness_score rules_html supported short_summary],
+        %i[id name hotness_score rules_html supported short_summary],
```

Comment:
This value changing from strings to symbols is a non-issue since there's one more step before we return the tags to the UI (there's another step making the JSON output)?

Reply:
Exactly! The hash is then converted to JSON by this:

https://github.com/forem/forem/blob/f2224979132b1a0815bc74f784a9aa6d6f75db57/app/controllers/search_controller.rb#L53-L59

we test it here: https://github.com/forem/forem/blob/f2224979132b1a0815bc74f784a9aa6d6f75db57/spec/requests/search_spec.rb#L33